lib/boot.rb

```diff
@@ -37,54 +52,25 @@ module Dor
 #   end
 # end
 
-# Sdr::Config contains the constants that are required within this project
-module Sdr
-  Config = Confstruct::Configuration.new do
-    ingest_transfer do
-      account "userid@dor-host.stanford.edu"
-      export_dir "/dor/export/"
-    end
-    logdir File.join(ROBOT_ROOT, 'log')
-    migration_source "/services-disk/sdr2objects"
-    sdr_recovery_home nil
-    enqueue_max 10
-    audit_verbose false
-  end
-end
-
-
 require 'druid-tools'
-require 'dor/services/workflow_service'
 require 'lyber_core/robot'
 require 'lyber_core/log'
 require 'moab_stanford'
 include Stanford
 require 'sdr_replication'
 
-# Load the environment file based on Environment.  Default to local
+# Load the environment file.  The environment config file should
+# override the default configurations above.
 environment = ENV['ROBOT_ENVIRONMENT'] || 'development'
 require File.join(ROBOT_ROOT,"config/environments/#{environment}")
 
-require 'sdr/sdr_robot'
+require 'dor/services/workflow_service'
+log_file = File.join(ROBOT_ROOT, 'log', 'workflow_service.log')
+wfs_logger = Logger.new(log_file, 'weekly')
+wfs_options = {
+  logger: wfs_logger
+}
+Dor::WorkflowService.configure Dor::Config.workflow.url, wfs_options
 
-module Dor
-  module WorkflowService
-    class << self
-      # @param [String] url points to the workflow service
-      # @param [Hash] opts optional params
-      # @option opts [String] :client_cert_file path to an SSL client certificate
-      # @option opts [String] :client_key_file path to an SSL key file
-      # @option opts [String] :client_key_pass password for the key file
-      def configure(url, opts={})
-        params = {}
-        params[:ssl_client_cert] = OpenSSL::X509::Certificate.new(File.read(opts[:client_cert_file])) if opts[:client_cert_file]
-        params[:ssl_client_key]  = OpenSSL::PKey::RSA.new(File.read(opts[:client_key_file]), opts[:client_key_pass]) if opts[:client_key_file]
-        params[:timeout] = 120
-        params[:open_timeout] = 120
-        @@resource = RestClient::Resource.new(url, params)
-      end
-    end
-  end
-end
+require 'sdr/sdr_robot'
 
-Dor::WorkflowService.configure Dor::Config.workflow.url
```

Comment:
All the `Dor::WorkflowService` changes are substantial and _could_ break the `master` branch functionality; so very concerned about testing this on the -dev system.